railties/lib/rails/generators/rails/app/templates/config/databases/ibm_db.yml

```diff
@@ -11,14 +11,14 @@
 #       export IBM_DB_LIB=/opt/ibm/db2/V9.7/lib32
 #       gem install ibm_db
 #
-#   On Mac OS X 10.5:
+#   On macOS 10.5:
```

Comment:
I agree with @sikachu: There's no such thing as macOS 10.5/10.6.  That said, both of those are no longer receiving support.  If the 10.6 instructions still work on modern macOS, I'd recommend changing that to macOS without a version number, and removing the Mac OS X 10.5 section.